### PR TITLE
Port to 1.21

### PR DESCRIPTION
**Pull request by dima-dencep**

### CODE REVIEW

File: minecraft/forge/src/main/java/dev/kosmx/playerAnim/impl/neoforge/HelperImpl.java [hunk 1]
```diff
@@ -1,24 +1,11 @@
...
-
-    private static boolean hasClass(String name) {
-        try {
-            // This does *not* load the class!
-            MixinService.getService().getBytecodeProvider().getClassNode(name);
-            return true;
-        } catch (ClassNotFoundException | IOException e) {
-            return false;
-        }
+        return LoadingModList.get().getModFileById("bendylib") != null;
```
Comment (KosmX) [inline minecraft/forge/src/main/java/dev/kosmx/playerAnim/impl/neoforge/HelperImpl.java:9]: Initially I was trying to get mod from API, but that version of forge simply breaks if I do this before forge is ready.

Is this solution safe to access from early entry?

Comment (dima-dencep) [reply minecraft/forge/src/main/java/dev/kosmx/playerAnim/impl/neoforge/HelperImpl.java:9]: Yes, it's a **Loading**ModList, not a ModList (there are two in FML).

**Comment by KosmX** [comment]
I've already reimplemented the update, just forgot to push

File: minecraft/forge/src/main/java/dev/kosmx/playerAnim/impl/neoforge/HelperImpl.java (hunk 1 above)
Comment (KosmX) [reply minecraft/forge/src/main/java/dev/kosmx/playerAnim/impl/neoforge/HelperImpl.java:9]: Okay, I'll verify if it works correctly and if it does, I'll use this

**Pull request closed without merging**